addon/bootstrap.js

```diff
@@ -7,8 +7,8 @@ function startup(data, reason) {
   });
 }
 
-function install() {
-}
+function shutdown(data, reason) {}
```

Comment:
Q: Do we need to remove the listener that we added in `startup()` above during one of these lifecycle functions?